packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -237,6 +237,11 @@ export type TableProps<
    * @beta
    */
   enableVirtualization?: boolean;
+  /**
+   * When using virtualized table, initialize the table scrolled to this item
+   * Does not support non-virtualized tables or tables with pagination
+   */
+  scrollToItem?: T;
```

Comment:
hm... Would we add support for all table types later on? Would it make sense?

Cause otherwise, not sure I am a fan of prop, which works only with virtualization is enabled :/

Reply:
For non-virtualized tables, we could try an approach like [your workaround](https://codesandbox.io/s/itwinui-react-table-scrollintoview-ol1z9x?file=/src/App.tsx:1270-1319); we'd need to add the ref to a row instead of a cell, I think. Or there may be a simpler solution.

We could probably hack together a solution for paginated tables, though I'm not too familiar with that part of the code. I suppose you can use pagination with and without virtualization?

_Alternatively_, we could update the virtualization prop to take an object, and include the scroll to item as an optional property. That would solve the problem of the prop only working with virtualization, though it isn't as logical to me.

Reply:
yeah, so pagination can be used with virtualization, tehcnically :D (@bentleyvk)
Maybe then we can have a separate prop, just mark it `@beta` as virtulization.. If we plan to add this scrolling to index with all the cases in future...
Need to think a little here.
@bentleyvk @mayank99 opinions?

Reply:
I think it makes sense to fallback to native `scrollIntoView` for non-virtualized tables. Adding paginated support will get complicated so I think we should leave that to the user because they can easily find out what page the row is in.

Reply:
Well ok.. Lets leave the prop then.

Reply:
I added support for non-virtualized tables, but left off support for pagination for now. Since it adds a bit more code, I wrapped up all the functionality into its own `useScrollToRow` hook.

The useMergedRefs hook allows us to add an extra ref on the row div, which can then be imperatively triggered to scroll to the specified row. It still allows users to add their own refs to the row, and the onRowInViewport callback is not affected.

I did not add support for paginated tables.